src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -440,6 +439,28 @@ private IAIState updateVisualState()
         job.setNameTag(this.getState().toString());
         //Update torch, seeds etc. in chestbelt etc.
         updateRenderMetaData();
+
+        if ( getOwnBuilding().hasWorkerOpenRequests( worker.getCitizenData() ) )
```

Comment:
The hell is this formatting you’re doing ray? Spaces absolutely everywhere

Reply:
spaces, where? I can't see them

Reply:
Can’t tell if that’s sarcasm so here: 
![E9FD0090-18CC-4E9B-9B00-4BAFD872F569](https://user-images.githubusercontent.com/28527281/70574613-3c8a0280-1bf0-11ea-999d-382f2144cd8d.jpeg)

Reply:
Ahh those you mean =D Isn't that our default formatter?

Reply:
No it certainly isn’t lol. That’s completely abnormal